### ISSUE TITLE
IntegerDivisionByZeroException thrown by toString()

### ISSUE THREAD

**Issue by kibantony** [a14n/dart-decimal#13]
Sample code:
```
    Decimal foo = Decimal.parse('21.962962546543768');
    foo.toString(); // <-- throws IntegerDivisionByZeroException
```

Seems to only happen for values that with large scales. For example '21.1' works fine.

**Comment by kibantony** [comment]
Environment:
Dart VM version: 2.0.0-dev.43.0.flutter-52afcba357 (Fri Mar 30 20:33:12 2018 +0000) on "macos_x64"

**Comment by a14n** [comment]
Thanks for the report.

Fixed in version 0.2.0 (by upgrading to _rational-0.2.0_)
